discord/client.py

```diff
@@ -979,6 +979,40 @@ def fetch_guilds(self, *, limit=100, before=None, after=None):
         data = await self.http.create_guild(name, region, icon)
         return Guild(data=data, state=self._connection)
 
+    async def fetch_url_message(self, url: str):
```

Comment:
```suggestion
    async def fetch_message_from_jump_url(self, url: str):
```

I feel like this should be called `fetch_message_from_jump_url` (or something shorter), as though it is [`Message.jump_url`](https://github.com/Rapptz/discord.py/blob/d37839f38acd79d49bc19ae06d3067a455695e9b/discord/message.py#L471)

Reply:
I feel like `fetch_url_message` is decently well-fitting, and I can't think of a way to make a short enough name that isn't difficult to read. Adding "jump url" to the docstring should still allow anyone looking for the method to find it easily, and it shouldn't be too difficult for someone reading code to tell what "fetch_url_message" is doing.
Do you have any suggestions on how to improve the name without making it too long or sacrificing readability?

Reply:
`fetch_url_message` is weird (what's a url message?)
`fetch_message_url` is misleading (fetching the url?)
`fetch_message_from_url` should be fine without the `jump` imo.

Reply:
That works for me! @NCPlays how about you?

Reply:
I'm fine with that! :grinning: